src/mcode-pilot/mock-data/transformedDataModel.json

```diff
@@ -2,33 +2,114 @@
     "demographics":{
         "birthDate": {"type": "date", "values": [1920,1994], "weight": "uniform"},
         "gender": {"type": "choice", "values": ["male","female"], "weight": "uniform"},
-        "race": {"type": "choice", "values": ["White","Black","Unknown","Other unspecified (1991+)","Other (American Indian/AK Native; Asian/Pacific Islander)"],
-                                   "weight": [0.5,     0.2,    0.05,     0.01,                       0.24]}
+        "race": {"type": "choice", "values": [{"code":"2106-3", "codeSystem":"http://hl7.org/fhir/v3/Race", "displayName": "White", "codeSystemName":"cdc"},
```

Comment:
Just want to make sure I am correct in assuming that this file was only used for updating the mock data, it's not used for anything in the running application, correct?

Reply:
Yes this file is strictly for updating the mock data.